build.gradle

```diff
@@ -102,28 +93,80 @@ wrapper {
 }
 
 // BuildDate function adapted from http://stackoverflow.com/questions/19172565/how-append-date-build-to-versionnamesuffix-on-gradle
-def buildDate() {
+def buildDate = {
```

Comment:
Not saying this is wrong but why the change from a method to a closure?

Reply:
Because I think it is semantically more correct. In my opinion we use `buildDate` as a value and not as a method.